src/aspire/volume/volume.py

```diff
@@ -140,7 +147,9 @@ def astype(self, dtype, copy=True):
         :return: Volume instance
         """
         return self.__class__(
-            self.asnumpy().astype(dtype, copy=copy), symmetry_group=self.symmetry_group
+            self.asnumpy().astype(dtype, copy=copy),
+            pixel_size=self.pixel_size,
```

Comment:
Not saying we do this now, but if (when?) we have a third field that needs to be preserved across processing steps, we may want to to think of some more compact way to do this (`…, **self._metadata_dict` where the latter is a `@property` that rounds up the things we care about into a dict and returns it?).

Reply:
Sure, I'd be for abstracting that out in the future, maybe as it's own PR. A few classes could probably use some cleanup like that at this point.